zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/migration/DbMigratorImplTest.java

```diff
@@ -275,4 +290,83 @@ void shouldOnlyRunInitializationMigrationsWhenDbIsEmpty() {
     verify(migration).runMigration(any());
     verify(mockProcessingState.getMigrationState()).setMigratedByVersion(eq(CURRENT_VERSION));
   }
+
+  @Test
+  void shouldRunRoutingInfoInitializationWhenDesiredIsNotSet() {
```

Comment:
:wrench: I don't think these tests belong here. It should be in `RoutingInfoInitializationMigrationTest` or just add a test for `isInitialized` in `DbRoutingStateTest` because that is what we are essentially testing.